gitlab/v4/objects.py

```diff
@@ -2364,10 +2364,6 @@ class ProjectPipelineScheduleManager(CRUDMixin, RESTManager):
                      ('description', 'ref', 'cron', 'cron_timezone', 'active'))
 
 
-class ProjectPipelineJob(ProjectJob):
-    pass
-
-
```

Comment:
This has just been removed in another patch, along with the associated manager. Could you rebase your change?

Reply:
Merged from upstream.